aep/general/0134/aep.md.j2

```diff
@@ -0,0 +1,321 @@
+- The request message **must** exactly match the RPC name, with a `Request`
+  suffix.
+- The response message **must** be the resource itself.
+  - The response **should** include the fully-populated resource, and **must**
+    include any fields that were sent and included in the update mask unless
+    they are input only (see AEP-203).
+  - If the update RPC is [long-running](#long-running-update), the response
+    message **must** be an `Operation` for which the return type is the
+    resource itself.
+- The method **should** support partial resource update, and the HTTP verb
+  **should** be `PATCH`.
+
+{% tab proto %}
+
+```proto
+rpc UpdateBook(UpdateBookRequest) returns (Book) {
+  option (google.api.http) = {
+    patch: "/v1/{book.path=publishers/*/books/*}"
+    body: "book"
+  };
+  option (google.api.method_signature) = "book,update_mask";
+}
+```
+
+- The resource's `path` field **should** map to the URI path.
+  - The `{resource}.path` field **should** be the only variable in the URI
+    path.
+- There **must** be a `body` key in the `google.api.http` annotation, and it
+  **must** map to the resource field in the request message.
+  - All remaining fields **should** map to URI query parameters.
+- There **should** be exactly one `google.api.method_signature` annotation,
+  with a value of `"{resource},update_mask"`.
+
+**Note:** Unlike the other four standard methods, the URI path here references
+a nested field (`book.path`) in the example. If the resource field has a word
+separator, `snake_case` is used.
+
+{% tab oas %}
+
+**Note:** OAS example not yet written.
+
+{% endtabs %}
+
+### Request message
+
+Update methods implement a common request message pattern:
+
+- The request message **must** contain a field for the resource.
+  - The name of this field **must** be the singular form of the resource's
+    name.
+- The request message **must not** contain any required fields other than those
+  described in this section, and **should not** contain other optional fields
+  except those described in this or another AEP.
+
+{% tab proto %}
+
+```proto
+message UpdateBookRequest {
+  // The book to update.
+  //
+  // The book's `path` field is used to identify the book to update.
+  // Format: publishers/{publisher}/books/{book}
+  Book book = 1 [(google.api.field_behavior) = REQUIRED];
+
+  // The list of fields to update.
+  google.protobuf.FieldMask update_mask = 2;
+}
+```
+
+- The request message field for the resource **must** map to the `PATCH` body.
+- The request message field for the resource **should** be [annotated as
+  required][aep-203].
+  - A `path` field **must** be included in the resource message. It **should**
+    be called `path`.
+  - The field **must** identify the [resource type][aep-123] of the resource
+    being updated.
+- If partial resource update is supported, a field mask **must** be included.
+  It **must** be of type `google.protobuf.FieldMask`, and it **must** be called
```

Comment:
@mkistler  @hudlow more of an FYI, but I think @rofrankel and I agree that we may want to adopt a better patch language.

see https://github.com/aep-dev/aep.dev/issues/109.

Reply:
Agreed, but marking resolved since it's beyond the scope of this PR (and this will eventually block merging the PR if open).